File: ListView/app/src/main/java/com/hfad/listview/MainActivity.java
package com.hfad.listview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.text.Editable;
import android.text.TextWatcher;
import android.util.Log;
import android.widget.ArrayAdapter;
import android.widget.EditText;
import android.widget.ListView;
import android.widget.Toast;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    private static final String TAG = "MainActivity";

    private ArrayAdapter adapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ListView list = (ListView) findViewById(R.id.theList);
        Log.d(TAG, "onCreate: Started.");

       final ArrayList<String> names = new ArrayList<>();
        names.add("One");
        names.add("Two");
        names.add("Three");
        names.add("Four");
        names.add("Five");

        adapter = new ArrayAdapter(this, R.layout.list_item_layout, names);
        list.setAdapter(adapter);

        list.setOnClickListener((adapterView, view, i, 1)   {
            Log.d(TAG, "onItemClick: name: " + names.get(i));
            Toast.makeText(MainActivity.this, "You clicked on: " + names.get(i), Toast.LENGTH_SHORT).show();
        });



    }
}

File: Lesson_6/app/src/main/java/com/hfad/lesson_6/MainActivity.kt
package com.hfad.lesson_6

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.ListView
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    private var listView: ListView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
listView = findViewById(R.id.listView)
        var nameList = ArrayList<String>()
        nameList.add("Иван")
        nameList.add("Серега")
        nameList.add("Павел")
        nameList.add("Толя")
        nameList.add("Коля")
        nameList.add("Зоя")


        val adapter = ArrayAdapter(this, android.R.layout.simple_list_item_1, nameList)

            listView?.adapter = adapter
        listView?.setOnItemClickListener{}
        }

}